tests/containers/container_engine.pm

```diff
@@ -111,7 +111,7 @@ sub basic_container_tests {
     assert_script_run("$runtime image rm example.com/tw-commit_test");
 
     ## Test connectivity inside the container
-    assert_script_run("$runtime container exec basic_test_container curl -sfI https://opensuse.org", fail_message => "cannot reach opensuse.org");
+    script_retry("$runtime container exec basic_test_container curl -sfIL https://opensuse.org", retry => 3, delay => 5, fail_message => "cannot reach opensuse.org");
```

Comment:
it would be better to use `conncheck.opensuse.org` as it was created for this purpose

Reply:
I would also increase the delay to 60. If it fails now, give the network some time before trying again